pkgs/games/dynablaster-revenge/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv
+, lib
+, mkDerivation
+, fetchFromGitHub
+, qtbase
+, qmake
+, wrapQtAppsHook
+, libGLU
+, SDL2
+, alsa-lib
+
+, isServer ? false
+}:
+
+mkDerivation rec {
+  pname = "dynablaster-revenge"
+    + lib.optionalString isServer "-server";
+  version = "unstable-2021-01-02";
+
+  src = fetchFromGitHub {
+    owner = "varnholt";
+    repo = "dynablaster_revenge";
+    rev = "75538ac3f1080fbdf91c58b533e487ceb6eb2aec";
+    sha256 = "0qpdq8x3sjwgrqrfyfjwsf90rklf3lj0l0x7s8d0ss3yadq110la";
+  };
+
+  postUnpack = ''
+    export sourceRoot=$sourceRoot/${if isServer then "server" else "client"}
+  '';
+
+  postPatch = lib.optionalString isServer ''
```

Comment:
Does the file exist when not building the server? If so we should always patch to make things easier.

Reply:
Solved by the nice guy from upstream

Reply:
I think you closed the wrong conversation, this is about patching the server build script not its differing license :).

@SuperSandro2000 IMO it would be uglier to patch this unconditionally, either `${lib.optionalString !isServer "../server/"}server.pro` or replacing `sourceRoot` with `cd`ing.